activities/service/v1/notifications.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.v1";
+option go_package = "activities.v1";
+
+import "google/protobuf/empty.proto";
+import "activities/domain/v1/domain.proto";
+import "google/protobuf/wrappers.proto";
+import "global/domain/v1/domain.proto";
+
+service notificationsService {
+  rpc updateNotificationChannelsConfiguration (UpdateNotificationChannelConfiguration) returns (google.protobuf.Empty);
+  rpc getNotificationChannelsConfiguration (google.protobuf.Empty) returns (NotificationChannelsConfigurationResponse);
+}
+
+message UpdateNotificationChannelConfiguration{
+  NotificationConfiguration configuration = 1;
+}
+message WebhookConfiguration{
+  string webhook_url = 1;
+  map<string, string> webhook_headers = 2;
+}
+
+message EmailConfiguaration{
+  string email = 1;
+}
+
+message NotificationChannelsConfigurationResponse{
+  repeated NotificationConfiguration configurations = 1;
+}
+
+message NotificationConfiguration{
```

Comment:
If I'm not mistaken in the service we keep only the Request and Response message. All the other entities are in a domain proto file.